src/components/Card.jsx

```diff
@@ -0,0 +1,35 @@
+const Card = ({ openSpots, coverImg, price, title, location, stats }) => {
+  let badgeText;
+  if (openSpots === 0) {
+    badgeText = "SOLD OUT";
+  } else if (location === "Online") {
+    badgeText = "ONLINE";
+  }
+
+  return (
+    <div className="w-44 text-xs flex flex-auto flex-col relative">
+      {badgeText && (
+        <div className="absolute top-2 left-2 bg-white py-1 px-2 rounded-sm font-bold">
+          {badgeText}
+        </div>
+      )}
+      <img
+        src={`/assets/${coverImg}`}
```

Comment:
why don't you import the all path for this image?

Reply:
I need more clarification on this comment.

Reply:
@ishoborabyose, instead of using the image name here, you can use the whole image path.

```js
<img src={coverImage} />
```